### ISSUE TITLE
AE2 disconnects often in the early late game

### ISSUE THREAD

**Issue by GamerHD286** [EnigmaticaModpacks/Enigmatica2Expert#2161]
### Modpack Version

1.90g

### Describe your issue.

I'm currently playing Enigmatica 2: Expert for a second time. I noticed an issue since I am in the early late game (Advanced Rocketry, Ender IO, etc.). It is that sometimes the Applied Energistics System isn't getting power for a few seconds. This happens sometimes after 4 hours, but sometimes every minute. It is really annoying when you must wait until your channels are active and I hope this issue will be fixed in the future!

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Both

### Discord Username

InvisChrisX

**Comment by tuca1234567890** [comment]
Do you have energy cells in your network?

**Comment by GamerHD286** [comment]
Yes, i have literally a Stack of Dense Energy Cells in my Network. Tomorrow I‘ll try to completely clean up and rebuild my system. This is not an easy task Ik But it‘s the last Option I have

**Comment by MommyHeather** [comment]
This isn't really a known bug outside of chunkloading or power issues - are you 100% certain that neither could be the case? (alternatively, are you using large amounts of toggle buses?)

**Comment by GamerHD286** [comment]
@MommyHeather What do you mean?

**Comment by GamerHD286** [comment]
@MommyHeather both points are proved. I guess I know what the problem is. P2Ps are really buggy. Because Youtubers like Lash are having problems with them too and the problems are the same.

**Comment by MommyHeather** [comment]
P2ps also don't really have any known bugs - one possible cause could be power issues if the p2ps are subnetted though?
